docs/conf.py

```diff
@@ -3,6 +3,13 @@
 # For the full list of built-in configuration values, see the documentation:
 # https://www.sphinx-doc.org/en/master/usage/configuration.html
 
+import os
+import sys
+
+# Add custom paths for modules
+sys.path.insert(1, os.path.abspath("../scripts"))
+sys.path.insert(2, os.path.abspath("../Surfactant"))
+
```

Comment:
Due to the ReadTheDocs yaml file now making so surfactant gets installed with the pip install "."  stuff, these lines shouldn't be needed.